creating-virtual-machines/virtualized-hardware-configuration.adoc

```diff
@@ -134,6 +134,43 @@ spec:
         claimName: myclaim
 ----
 
+Enabling cpu compatibility enforcement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable the cpu compatibility enforcement, user may expand the
+`feature-gates` field in the kubevirt-config config map by adding the
+`CPUNodeDiscovery` to it.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kubevirt
+  labels:
+    kubevirt.io: ""
+data:
+  feature-gates: "CPUNodeDiscovery"
+...
+----
+
+This feature-gate allows kubevirt to take VM cpu model and cpu features and create node selectors from them.
+With these node selectors, VM can be scheduled on the node which can support VM cpu model and features.
+
+To properly label the node, user can use (only for cpu models)
+https://github.com/ksimon1/cpu-node-labeller[cpu-node-labeller] in combination with
```

Comment:
We should not link to personal repositories. What about proposing this and the following below to move to kubevirt org?

Reply:
@fabiand good idea